starters/next13-zustand-bulma/README.md

```diff
@@ -1,38 +1,117 @@
+  - [Demo Implementation](#demo-implementation)
+  - [Deployment](#deployment)
+
+</details>
+
+## Tech Stack
+
+- [Next.js v13.x](https://nextjs.org)
+- [React v18.x](https://reactjs.org)
+- [Zustand v4.x](https://react-query.tanstack.com/)
+- [Bulma](https://bulma.io/)
+
+#### Included Tooling
+
+- [Jest](https://jestjs.io/) - Test runner
+- [Typescript v4](https://www.typescriptlang.org/) - Type checking
+- [Storybook v7](https://storybook.js.org/) - Component sandbox and docs
+- [ESLint](https://eslint.org/) - Code linting
+- [Prettier](https://prettier.io/) - Code formatting
+
+## Getting Started
+
+- start by [creating a new Starter Kit](SETUP.md) project
+- `cd` into your project directory and run `npm install`.
+- Run `npm run dev` to start the development server.
+- Open your browser to `http://localhost:3000` to see the included example code running.
+
+## Available Commands
+
+- `npm run dev` - Starts the development server.
+- `npm run test` - Runs the unit tests.
+- `npm run storybook` - Starts the Storybook UI.
+- `npm run lint` - Runs ESLint on the project.
+- `npm run prettier` - Formats code for the entire project
+
+## Project Details
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+### Kit Organization / Architecture
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+This Kit uses new Next13 features, such as [`AppDir` router](https://beta.nextjs.org/docs/routing/fundamentals#the-app-directory) and [server components](https://beta.nextjs.org/docs/rendering/server-and-client-components).
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+We've chose to split out a `components` directory for interactive UI components, and a `store` directory for the Zustand stores used for both global and some component state management.
+
+This allows us to keep the `app` directory focused on routing, layouts, and loading data, while passing that data to the more interactive components with more complex needs.
+
+### Styling and Theme
+
+The project uses [Bulma](https://bulma.io/) as the main styling framework. Bulma offers a way to "theme" their components and utility classes by overriding their default SASS variables.
+
+Overrides are done in the `base-styles.scss` file, which is imported in the `src/app/global.scss` file. Bulma components and utility classes can be added to components as needed as they are included in the `global.scss` file, which in turn is imported in the root layout.
+
+You should imoprt the `base-styles.scss` file in any component `*.module.scss` file, as per the example in `app/counter-example.page.tsx` to have access to the [Bulma variables]() for color and spacing in your component.
```

Comment:
```suggestion
You should import the `base-styles.scss` file in any component `*.module.scss` file, as per the example in `app/counter-example.page.tsx` to have access to the [Bulma variables]() for color and spacing in your component.
```

Reply:
Also, it looks like the Bulma variables link doesn't link anywhere

Reply:
Bulma link fixed in 2d02ec6